### PR TITLE
GH-1768 Localize retrospective interval options

### LINKED ISSUE #1768: Localize retrospective interval options
The interval for the retrospective is not localized like the status update:

image

_Issue created from a [Mattermost message](https://community.mattermost.com/_redirect/pl/fm755bo6s3f78m1f7eryubcx8r) *by @m-ripper*._

### LINKED ISSUE #1770: Auto-follow runs button not localized
It seems the "Auto-follow runs" button is not localized.

_Issue created from a [Mattermost message](https://community.mattermost.com/_redirect/pl/tugp3dbzt3db5pkky8pukz6ztw) *by @m-ripper*._

### PR DESCRIPTION
**Pull request by m-ripper**
## Summary

Localize retrospective interval options and "Auto-follow runs"-button.

## Ticket Link

Fixes #1768 
Fixes #1770 

## Checklist

- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import React, {useMemo} from 'react';
+import {Duration} from 'luxon';
+
+import {Mode, Option, useMakeOption} from 'src/components/datetime_input';
+import {StyledSelect} from 'src/components/backstage/styles';
+
+interface Props {
+    seconds: number;
+    setSeconds: (seconds: number) => void;
```
Comment (calebroseland) [inline webapp/src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector.tsx:14]: nit: onChange would be more accurate
```suggestion
    onChange: (seconds: number) => void;
```

File: webapp/src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    disabled?: boolean;
+
+}
+const RetrospectiveInterval = (props: Props) => {
```
Comment (calebroseland) [inline webapp/src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector.tsx:18]: nit: slightly better clarity
```suggestion
const RetrospectiveIntervalSelector = (props: Props) => {
```

File: webapp/src/components/backstage/playbook_editor/outline/section_retrospective.tsx [hunk 3]
```diff
@@ -10,23 +10,16 @@ import {FullPlaybook, Loaded, useUpdatePlaybook} from 'src/graphql/hooks';
 import {Metric, PlaybookWithChecklist} from 'src/types/playbook';
 import {SidebarBlock} from 'src/components/backstage/playbook_edit/styles';
 import Metrics from 'src/components/backstage/playbook_edit/metrics/metrics';
-import {BackstageSubheader, BackstageSubheaderDescription, StyledSelect} from 'src/components/backstage/styles';
+import {BackstageSubheader, BackstageSubheaderDescription} from 'src/components/backstage/styles';
 import MarkdownEdit from 'src/components/markdown_edit';
 import {savePlaybook} from 'src/client';
+import RetrospectiveInterval from 'src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector';
```
Comment (calebroseland) [inline webapp/src/components/backstage/playbook_editor/outline/section_retrospective.tsx:16]:
```suggestion
import RetrospectiveIntervalSelector from 'src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector';
```

**Review by calebroseland — APPROVED** [review]
Very nice 👨‍🍳🤌. 2 minor nits, but feel free to skip them.

**Review by gbochora — APPROVED** [review]
Looks good!

**Pull request merged**